src/types/drupal/node.ts

```diff
@@ -127,6 +127,13 @@ export interface NodeFaqMultipleQA extends NodeAbstractResource {
   field_buttons_repeat: boolean
 }
 
+export type FacilityOperatingStatusFlags =
+  | 'normal'
+  | 'notice'
+  | 'limited'
+  | 'closed'
+  | string // TODO: Remove the catch-all; needed right now for importing JSON mock data; can remove after we implement runtime data validation
```

Comment:
It's actually pretty annoying that I still need `| string` on this. Makes this union kinda silly because the static type gets collapsed to `string`, and we don't get even get any auto-fill suggestions from our language server. Still, it's good to call out the expected values. 🤷